frameworks/digital-outcomes-and-specialists/questions/services/securityConsultant.yml

```diff
@@ -12,3 +12,5 @@ questions:
   - securityConsultantDayRate
 
 empty_message: You haven't added a cyber security consultant
+
+hint: Minimise the chance of data or information systems security breaches. Ensure information is protected against unauthorised or unintended access. Put systems in place to prevent data destruction or disruption.
```

Comment:
This seems oddly phrased for a Security Consultant as a person. It made sense for the "Security" service type, but doesn't seem quite right for a person.